frontend/src/pages/FAQ.vue

```diff
@@ -378,8 +378,8 @@
               <template v-slot:mainnet>
                 <a :href="getEtherscanUrl(deployments.registry, 1)" class="hyperlink" target="_blank"> mainnet </a>
               </template>
-              <template v-slot:goerli>
-                <a :href="getEtherscanUrl(deployments.registry, 4)" class="hyperlink" target="_blank"> Goerli </a>
+              <template v-slot:sepolia>
+                <a :href="getEtherscanUrl(deployments.registry, 4)" class="hyperlink" target="_blank"> Sepolia </a>
```

Comment:
same
```suggestion
                <a :href="getEtherscanUrl(deployments.registry, 11155111)" class="hyperlink" target="_blank"> Sepolia </a>
```

Reply:
Fixed in the same commit above: https://github.com/ScopeLift/umbra-protocol/pull/521/commits/37be3194d7c9d2dc67f5db05b47395ebeb1f4b8e